Dockerfile

```diff
@@ -3,14 +3,15 @@
 # upgrade everything by simply updating the Alpine version.
 #
 # Alpine is updated every 6 months so all packages are pretty recent.
-FROM alpine:3.10
+FROM alpine:3.12
 
 RUN apk update && apk add --no-cache \
   chromium \
   git \
   make \
   nodejs \
   npm \
```

Comment:
I'm not sure if we still need npm here. I note that it still exists in the client's Dockerfile.

Reply:
I confirm `npm` it is not needed.

In the client I see a use of  `npm publish` command, which could be modified to `yarn publish`